js/cryptocom.js

```diff
@@ -71,65 +71,105 @@ module.exports = class cryptocom extends Exchange {
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/147792121-38ed5e36-c229-48d6-b49a-48d05fc19ed4.jpeg',
                 'test': 'https://uat-api.3ona.co/v2',
-                'api': 'https://api.crypto.com/v2',
+                'api': {
+                    'spot': 'https://api.crypto.com/v2',
+                    'derivatives': 'https://deriv-api.crypto.com/v1',
+                },
                 'www': 'https://crypto.com/',
                 'referral': '',
                 'doc': 'https://exchange-docs.crypto.com/',
                 'fees': 'https://crypto.com/exchange/document/fees-limits',
             },
             'api': {
-                'public': {
-                    'get': {
-                        'public/auth': 1,
-                        'public/get-instruments': 1,
-                        'public/get-book': 1,
-                        'public/get-candlestick': 1,
-                        'public/get-ticker': 1,
-                        'public/get-trades': 1,
-                        'public/margin/get-transfer-currencies': 1,
-                        'public/margin/get-load-currenices': 1,
-                        'public/respond-heartbeat': 1,
+                'spot': {
+                    'public': {
+                        'get': {
+                            'public/auth': 1,
+                            'public/get-instruments': 1,
+                            'public/get-book': 1,
+                            'public/get-candlestick': 1,
+                            'public/get-ticker': 1,
+                            'public/get-trades': 1,
+                            'public/margin/get-transfer-currencies': 1,
+                            'public/margin/get-load-currenices': 1,
+                            'public/respond-heartbeat': 1,
```

Comment:
We could probably remove the `public/` and `private/` prefixes from these urls, cause the implicit methods get names like spotPublicGetPublicGetTicker, too much of PublicGet, no?

Reply:
i don't thin

> > so if you can download the app get verified then send me ur apiKeys I can finish the private api for the derivates
> 
> I'm in the verification process, will send you the keys when i have them.

please update me when you get it

Reply:
if we remove them we'll need to have more string concatenation

Reply:
Yes, but we do the same everywhere anyway. Up to you.